src/main/public/stylesheets/components/_form.scss

```diff
@@ -16,3 +16,10 @@
     @include button($grey-3);
```

Comment:
can't comment on the image, but do we not already have it in the app for the amountBreakdown page?

Reply:
we have two different approaches:

```.input-symbol-pound::before {
    content: '£';
    left: 7px;
    position: absolute;
    top: 1px;
}```

and this image.

I'll talk to UX guys to find out if this is what we want. If not - I'll align it.

Reply:
Lets not use an image, can't see any reason why image would be better then single pound char.

Reply:
I have talked to Tarik and we should use css solution, not background img. Updated.